### PR TITLE
Add maxRetries to submit wdl tasks

### PR DESCRIPTION
**Pull request by samanehsan**
Note this adds maxRetries to all submit wdl tasks except for the create submission task, since it is not currently idempotent. 

After merging, configure the `max_cromwell_retries` parameter in Lira to override the defaults in pipeline-tools and use cromwell task retries 

Please ensure the following when opening a PR:
- [ ] Added or updated tests
- [ ] Updated documentation
- [ ] Applied style guidelines
- [ ] Considered generalizability beyond our own use case

### CODE REVIEW

**Comment by samanehsan** [comment]
please retest this

**Comment by samanehsan** [comment]
retest this please

**Comment by rexwangcc** [comment]
Unfortunately, I forgot to change the version from `v0.28.0` to `v0.28.1` but tagged it with `v0.28.1` in a merged PR just now, so when you pre-merge this PR, can you change the docker tags to `v0.28.2`? Thanks!!

**Comment by samanehsan** [comment]
Sure @rexwangcc! I'm just looking into the failing mintegration test now 😞

**Comment by samanehsan** [comment]
retest this please

File: adapter_pipelines/submit.wdl [hunk 1]
```diff
@@ -9,6 +9,7 @@ task get_metadata {
   Int? individual_request_timeout
   Boolean use_caas
   Boolean record_http
+  Int max_retries = 0
```
Comment (rexwangcc) [inline adapter_pipelines/submit.wdl]: So how do we tweak the numbers here? we change the static input json or we pass it in from Lira?

Comment (samanehsan) [reply adapter_pipelines/submit.wdl]: It gets passed in from the Lira config

**Comment by samanehsan** [comment]
@rexwangcc the mintegration test is failing because of an issue with the test itself. Do you think it's okay to merge this?

**Comment by rexwangcc** [comment]
Yes! Go for it!

**Pull request merged**